uptraining_docai_processor_using_python/docai_uptraining.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            {"batch_input_config": {"gcs_prefix": {"gcs_uri_prefix": gcs_uri_prefix}}}
</pre><p>Change to use the built in class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>